Models/User.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace WsuTriangle.AttendanceTracker.Models;
+
+public class User
+{
+    public int UserId { get; set; }
+    public string UserName { get; set; } = string.Empty;
+    public string PasswordHash { get; set; } = string.Empty;
+    public string Role { get; set; } = "Member"; // Default to Member, could also be Admin
```

Comment:
If you plan on adding more roles, do so by making an Enum. Otherwise simply:

```suggestion
    public bool isAdmin { get; set; } = false;
```